apollo-router/src/configuration/mod.rs

```diff
@@ -437,6 +438,36 @@ impl Configuration {
             );
         }
 
+        // PQs.
```

Comment:
Agreed on both parts.  (My instinct would be that this would be something like allow_clients_to_register_operations to be concrete instead of marketing-y but probably that's not great given how long we've had that name.) Perhaps we do this as part of removing `preview_`.